### PR TITLE
v3.6.1 released - security patch

### PR DESCRIPTION
**Pull request by paulgear**
This is a strongly-recommended security update; see https://github.com/helm/helm/releases/tag/v3.6.1 and https://github.com/helm/helm/security/advisories/GHSA-56hp-xqp3-w2jf

### CODE REVIEW

**Review by diddledan — APPROVED** [review]
LGTM

**Comment by joedborg** [comment]
Thanks!

**Comment by Lin-Buo-Ren** [comment]
Do we still require 2 reviewers' approval before merging the MR?

**Pull request merged**
